modules/__tests__/Store-test.js

```diff
@@ -0,0 +1,59 @@
+jest.mock('../Dispatcher');
+
+import Dispatcher from '../Dispatcher';
+import Store, { StoreRegistry } from '../Store';
+
+describe('Store', () => {
+  const InitialState = [];
+
+  class DefaultStore extends Store {
+    getInitialState() {
+      return InitialState;
+    }
+  }
+
+  it('should have initial state', () => {
+    const store = new DefaultStore();
```

Comment:
Would it be a real use case (`new DefaultStore()`)?

Reply:
If we treat these tests as unit tests, we need things to be isolated. A real use case is about using a factory method `of(Dispatcher)` which does `new SomeStore()` and registers it in Dispatcher. There was no reason to register the store in this particular test, so I decided to just instantiate it.